src/frontend-pwa/src/views/BCServices/BCServices.tsx

```diff
@@ -35,7 +35,7 @@ export default function BCServices() {
   return (
     <ViewContainer>
       <ContentContainer>
-        {navigator.onLine
+        {navigator.onLine && !state.settings.offline_mode
```

Comment:
Can you explain the difference between how these are used?

Reply:
@BradyMitch navigator.onLine checks the browser to see if the app is online in general. So if the user wants to be online, but they don't have internet access, it will check for that. The state.settings one is a user setting that can be enabled and disabled within the app if the user wants to be offline to save data or something.